content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -54,32 +56,148 @@ Select **Confirm Export/Disconnect**
+
+The additional overhead of now-undersized existing blocks can be recovered by reading and rewriting existing data to the extended pool.
+This can occur naturally over the lifetime of the pool as ZFS rewrites data blocks when you modify or delete data.
+Alternately, you can manually replicate and rewrite the data to the pool.
+Data is rewritten using the extended block size and parity ratio.
+
+You can  use the [RAIDZ Extension Calculator]({{< relref "/References/ExtensionCalculator.md" >}}) to visualize potential lost headroom and capacity available to recover by rewriting existing data.
+
+For more information, see [Jim Salter's article](https://arstechnica.com/gadgets/2021/06/raidz-expansion-code-lands-in-openzfs-master/) at Ars Technica and the upstream [RAIDZ extension](https://github.com/openzfs/zfs/pull/15022) PR, sponsored by iXsystems, at OpenZFS.
+See also Louwrentius' ["ZFS RAIDZ Expansion Is Awesome but Has a Small Caveat"](https://louwrentius.com/zfs-raidz-expansion-is-awesome-but-has-a-small-caveat.html) for an in-depth discussion of lost capacity and recovering overhead.
+{{< /expand >}}
+
+To extend a RAIDZ VDEV, go to **Storage**.
+Locate the pool and click **Manage Devices** on the **Topology** widget to open the ***Poolname* Devices** screen.
+
+{{< trueimage src="/images/SCALE/Storage/DevicesExtend.png" alt="Devices Screen" id="Devices Screen" >}}
+
+Select the target VDEV and click **Extend** to open the **Extend Vdev** window.
+
+{{< trueimage src="/images/SCALE/Storage/ExtendVdev.png" alt="Extend Vdev" id="Extend Vdev" >}}
+
+Select an available disk from the **New Disk** dropdown menu.
+Click **Extend**.
+
+A job progress window appears.
+TrueNAS SCALE returns to the ***Poolname* Devices** screen when complete.
+
+### Adding a VDEV to a Pool
+
 ZFS supports adding VDEVs to an existing ZFS pool to increase the capacity or performance of the pool.
+To extend a pool by mirroring, you must add a data VDEV of the same type as existing VDEVs.
 
 {{< hint type=tip >}}
 You cannot change the original encryption or data VDEV configuration.
 {{< /hint >}}
 
+{{< expand "Adding VDEV Examples" "v" >}}
+* To make a striped mirror, add the same number of drives to extend a ZFS mirror.
+  For example, you start with ten available drives. Begin by creating a mirror of two drives, and then extending the mirror by adding another mirror of two drives. Repeat this three more times until you add all ten drives.
+* To make a stripe of two RAIDZ1 VDEVs (similar to RAID 50 on a hardware controller), add another three drives to extend the three-drive RAIDZ1.
+* To make a stripe of RAIDZ2 VDEVs (similar to RAID 60 on a hardware controller), add another four drives to extend the four-drive RAIDZ2.
```

Comment:
```suggestion
* To make a stripe of two 3-drive RAIDZ1 VDEVs (similar to RAID 50 on a hardware controller), add another three drives as a new RAIDZ1 VDEV to existing single 3-drive RAIDZ1 VDEV pool.
* To make a stripe of two 6-disk RAIDZ2 VDEVs (similar to RAID 60 on a hardware controller), add another six drives as a new RAIDZ2 VDEV to existing single 6-drive RAIDZ2 VDEV pool.
```
I'd propose to avoid word "extend" here to not confuse with RAIDZ expansion.  Also 4-disk RAIDZ2 is not the best configuration example, so I'd change it to 6 (or 10, if you like).